pyproject.toml

```diff
@@ -61,7 +61,9 @@ provision-virtual = [
     ]
 provision-container = []
 report-junit = [
-    "junit_xml>=1.9",
+    # Required to support XML parsing and checking the XSD schemas.
+    "lxml>=4.9.0",
+    "types-lxml",
```

Comment:
```suggestion
```
Typesheds should not be included. Still pending on https://github.com/teemtee/tmt/pull/2841 for how these will be handled in the future, but for now these are only manually tracked in pre-commit config

Reply:
Ok, I've excluded the `types-lxml` here, thanks.